src/Entities/InvoiceTax.php

```diff
@@ -11,22 +11,66 @@
 
 namespace Rebilly\Entities;
 
+use DomainException;
 use Rebilly\Rest\Resource;
 
 /**
  * Class InvoiceTax.
  */
 final class InvoiceTax extends Resource
 {
+    public const UNEXPECTED_CALCULATOR = 'Unexpected calculator. Only %s calculators are supported';
+
+    public const CALCULATOR_MANUAL = 'manual';
+
+    public const CALCULATOR_REBILLY_TAXJAR = 'rebilly-taxjar';
+
+    /**
+     * @return array
+     */
+    public static function allowedCalculators()
+    {
+        return [
+            self::CALCULATOR_MANUAL,
+            self::CALCULATOR_REBILLY_TAXJAR,
+        ];
+    }
+
     /**
      * @return string
      */
+    public function getCalculator()
+    {
+        return $this->getAttribute('calculator');
+    }
+
+    /**
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function setCalculator($value): self
```

Comment:
I noticed this is the only method with a return type declared.
Seems fine for the PHP version though!